sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -1645,7 +1644,9 @@ export interface SearchIndexerWarning {
 }
 
 // @public
-export interface SearchIndexingBufferedSender<T> {
+export class SearchIndexingBufferedSender<T> {
+    // Warning: (ae-forgotten-export) The symbol "IndexDocumentsClient" needs to be exported by the entry point index.d.ts
```

Comment:
missing export - when you export it, could you also declare that `SearchClient` implements it, so we make sure the contract stays valid?

Reply:
Done